Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -87,6 +87,13 @@ def run
           We'd still love your contributions, though, so try another one
           that's not in the autobump list:
             #{Formatter.url("#{cask.tap.remote}/blob/master/.github/autobump.txt")}
+
+          If BrewTestBot doesn't open a version update pull request in the next 3
+          hours, it might be because there is an existing open pull request for
+          the #{cask.token} cask.
+
+          If this is the case, please open a version update pull request manually:
+            #{Formatter.url("https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#cask-related-pull-request")}
```

Comment:
Not your fault but: this is way too much text to output here. My experience is the more text you output in these sort of cases, the less likely people are to read any of it. I'd rather we fixed the underlying issues here rather than improved our messaging around them.

Reply:
@carlocab Yeh, fair, just think https://github.com/Homebrew/brew/pull/18206 may be a better fit here. Let me know what you think.

Reply:
Thanks; will have a look later today!